rb/lib/selenium/webdriver/common/error.rb

```diff
@@ -153,7 +164,7 @@ class IMENotAvailableError < WebDriverError; end # 30
       class IMEEngineActivationFailedError < WebDriverError; end # 31
 
       #
-      # Argument was an invalid selector (e.g. XPath/CSS).
+      # Argument was an invalid selector.
```

Comment:
@twalpole They were changed to match the descriptions in the [WebDriver spec](https://w3c.github.io/webdriver/webdriver-spec.html#handling-errors) as per @titusfortner 's direction.